README.md

```diff
@@ -1 +1,60 @@
-## AoE4 World Stream Overlay
+# AoE4 World Overlay
+### A tool for streamers to display information about ongoing games in their broadcasts.
+
+* Supports all all Ranked and Quickmatch modes (1v1, 2v2, etc)
+* Shows players' names, ranking, civ and win rate
+* Inlcudes all new civilizations
+* Ready to use as a browser source in OBS and Streamlabs OBS
+
+<img width="1005" alt="CleanShot 2022-11-13 at 02 08 30@2x" src="https://user-images.githubusercontent.com/6642554/201501074-ea967231-59cb-44b4-b339-437bf741255c.png">
+
+
+> **Warning** 
+> This is a beta release and intended for people excited to use and test the overlay while we finilaze the features and iron out any bugs. The url you use to add the overlay source will change in the future and we may make slight changes to the appearance and behavior of the overlay until then.
+> We highly recommend you join our #overlay channel on our [Discord](https://discord.gg/aoe4) to stay up to date on new releases and changes. It's also a good place to share feedback or get help.
+
+---
+
+### How to use
+The overlay is actually a personalized web page that you can include as a browser source in your streaming software. First, you will need to create the right url, outlined below, then you can add it as a source.
+
+#### Profile ID
+The url for your overlay needs your AoE4 profile ID. Your profile ID is the number in the url of your profile page on aoe4world.com. For example, coRe's profile url is https://aoe4world.com/players/7090781-coRe, so his profile ID is 7090781. 
+
+### Default url
+To use the overlay in the default setup, replace the numbers in the below url with your profile ID
+
+```
+https://overlay-beta.aoe4world.com/?theme=top&includeAlts=true&profileId=1234567
+```
+
+> **Note** This url will change once the overlay comes out of beta
+
+#### Default theme
+The default behavior is intended for showing the overlay at the top of your stream, centered to the screen. This way it overlays nicely with the current Age indicator. 
+
+#### Floating theme
+If you want to show the overlay in an arbitrary position, like the  corner of your stream, you can use the floating theme. This will render the overlay in a fixed width with round corners on all sides. To use the floating theme, replace `theme=top` with `theme=floating` in your url.
+
+#### Example
+```
+https://overlay-beta.aoe4world.com/?theme=floating&includeAlts=true&profileId=1234567
+```
+
+<img width="932" alt="Floating Theme" src="https://user-images.githubusercontent.com/6642554/201501063-67a07b01-474b-4460-9efe-5c701b6b436a.png">
+
+
+### Add as a browser source
+Once you have the url, you can add it as a browser source in your streaming software, following these guids:
+
+* [OBS](https://obsproject.com/wiki/Sources-Guide#browser-source)
+* [Streamlabs OBS](https://blog.streamlabs.com/introducing-browser-source-interaction-for-streamlabs-obs-d8fc4dcbb1fb)
+
+In both instances, use a width of at least 800px and a height of 400px. 
+
```

Comment:
Add `Note: Right click on the source and click Transform->Center Horizontally` to center the overlay.

That coz some people don't know how to do that and seeing it just off-center is soo annoying :smile: